ckanext/cnra_schema/plugins.py

```diff
@@ -12,7 +12,7 @@ class cnraSchema(SingletonPlugin):
     implements(ISpatialHarvester, inherit=True)
 
     def update_config(self, config):
-        toolkit.add_public_directory(config, "static")
+        toolkit.add_resource('fanstatic', 'cnra')
```

Comment:
Can you rename the reference for the fanstatic directory from `cnra` to `cnra_schema`?
You will also need to update the resource import if you do.
```
{% resource 'cnra_schema/css/additional_metadata_form.css' %}
```
This will prevent overlapping imports from cnra theme.

Reply:
that's a good idea, I'll make this change